plugins/include/open62541/plugin/historydata/history_data_backend_memory.h

```diff
@@ -17,6 +18,9 @@ _UA_BEGIN_DECLS
 UA_HistoryDataBackend UA_EXPORT
 UA_HistoryDataBackend_Memory(size_t initialNodeIdStoreSize, size_t initialDataStoreSize);
 
+UA_HistoryDataBackend UA_EXPORT
```

Comment:
Add a short documentation to the public API.
What is the meaning of the arguments?